<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>DBoxFE_GameSettings</class>
 <widget class="QWidget" name="DBoxFE_GameSettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Gamepreferences</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../res/dboxfe.qrc" >:/pics/images/default.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Gameoption</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="1" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4" >
          <property name="minimumSize" >
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>110</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Compatibility:</string>
          </property>
          <property name="buddy" >
           <cstring>LEGameCompatibility</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LEGameCompatibility" />
        </item>
       </layout>
      </item>
      <item row="0" column="1" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="minimumSize" >
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>110</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Year:</string>
          </property>
          <property name="buddy" >
           <cstring>dateEditGameYear</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LEGamYear" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkBoxDate" >
          <property name="minimumSize" >
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Add date?</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="checked" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label" >
          <property name="minimumSize" >
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>110</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Category:</string>
          </property>
          <property name="buddy" >
           <cstring>cbxGameCategory</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbxGameCategory" >
          <item>
           <property name="text" >
            <string>Adventure</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Brick</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Fight</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Flight</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Misc</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Platform</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Progs</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Puzzle</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Race</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>RPG</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Spaceshooter</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Sport</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Strategy</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>War</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_6" >
          <property name="minimumSize" >
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>110</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Game:</string>
          </property>
          <property name="buddy" >
           <cstring>LEGamYear</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LEGame" />
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="minimumSize" >
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>110</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Developer:</string>
          </property>
          <property name="buddy" >
           <cstring>LEGamYear</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LEGameDeveloper" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QTreeWidget" name="twGameSettings" >
       <property name="contextMenuPolicy" >
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="rootIsDecorated" >
        <bool>false</bool>
       </property>
       <property name="itemsExpandable" >
        <bool>false</bool>
       </property>
       <column>
        <property name="text" >
         <string>Game</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Category</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Developer</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Year</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Compatibility</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>DosBox version</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Link</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="gameImage" >
       <property name="minimumSize" >
        <size>
         <width>230</width>
         <height>240</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>230</width>
         <height>240</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="scaledContents" >
        <bool>true</bool>
       </property>
       <property name="buddy" >
        <cstring>twGameSettings</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnSave" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnImage" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Image</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>21</width>
         <height>26</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnChange" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Change</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPreview" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Preview</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>79</width>
         <height>26</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdd" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemove" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>26</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>twGameSettings</tabstop>
  <tabstop>LEGame</tabstop>
  <tabstop>cbxGameCategory</tabstop>
  <tabstop>LEGameDeveloper</tabstop>
  <tabstop>LEGameCompatibility</tabstop>
  <tabstop>btnImage</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnPreview</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>btnRemove</tabstop>
  <tabstop>btnChange</tabstop>
 </tabstops>
 <resources>
  <include location="../res/dboxfe.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>chkBoxDate</sender>
   <signal>toggled(bool)</signal>
   <receiver>LEGamYear</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>874</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel" >
     <x>664</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkBoxDate</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_5</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>881</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel" >
     <x>568</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
